desktop/src/main/java/bisq/desktop/main/settings/preferences/PreferencesView.java

```diff
@@ -97,14 +98,22 @@
 @FxmlView
 public class PreferencesView extends ActivatableViewAndModel<GridPane, PreferencesViewModel> {
 
+    private final Preferences preferences;
+    private final FeeService feeService;
+    //private final ReferralIdService referralIdService;
+    private final AssetService assetService;
+    private final FilterManager filterManager;
+    //private ComboBox<BaseCurrencyNetwork> selectBaseCurrencyNetworkComboBox;
+    private final DaoFacade daoFacade;
+    private final BSFormatter formatter;
+    private final boolean daoOptionsSet;
+    private final boolean displayStandbyModeFeature;
```

Comment:
please do not commit non-changes. makes it hard to review and introduces unnecessary big change sets.

Reply:
I don't understand.   Are the non-changes you mentioned references to the comments (lines 103, 106)?  Those were there from previous commits (before this PR).  I did not add these comments.  Should I delete them?

Or, are you referring to the Intellij code reformatting which rearranged the order of field definitions?  I have been reformatting files I touch, using the IDE code style versioned in git.  Should I not do that?

Reply:
> Or, are you referring to the Intellij code reformatting which rearranged the order of field definitions? I have been reformatting files I touch, using the IDE code style versioned in git. Should I not do that?

that is what I meant. please only commit changes that have something to do with the topic of your PR. Thanks!